Dockerfile

```diff
@@ -4,8 +4,6 @@ FROM registry.access.redhat.com/ubi8/ubi AS builder
 
 RUN yum install -y ca-certificates git go-toolset make
 ENV PATH="/root/go/bin:${PATH}"
-ADD https://password.corp.redhat.com/RH-IT-Root-CA.crt /etc/pki/ca-trust/source/anchors/
-RUN update-ca-trust extract
```

Comment:
This shouldn't have been removed just to mitigate the failing tests, instead you could try doing:
```
RUN curl -sfL https://password.corp.redhat.com/RH-IT-Root-CA.crt \
    -o /etc/pki/ca-trust/source/anchors/RH-IT-Root-CA.crt ;\
    update-ca-trust
```